can_tools/scrapers/base.py

```diff
@@ -617,6 +617,9 @@ def find_unknown_variable_id(self, engine: Engine, df: pd.DataFrame):
 
     def find_unknown_location_id(self, engine: Engine, df: pd.DataFrame):
         locs = pd.read_sql("select * from locations", engine)
+        if "location" in df.columns:
+            good_rows = df.location.isin(locs.location)
+            return df.loc[~good_rows, :]
```

Comment:
Hrm.  The below (`location_name`) version of this code seems to filter to locations in `self.state_fips` so I'm wondering if we should do the same for this new `location` version?

That said I actually don't see `state_fips` defined / used on this class so I'm a bit confused. Not sure where it comes from. :/

Reply:
Yeah, this should probably either be a method of the [`StateDashboard`](https://github.com/covid-projections/can-scrapers/blob/main/can_tools/scrapers/official/base.py#L35) class (which has the `state_fips` attribute) or a stand-alone utils function. 

The `location_name` version specifies a state b/c states can have similarly named counties (eg, there are a million Washington counties). This limits the search to within the bounds of the state of the scraper (I think it was assumed that this would only be used on single state scrapers, but I'm not sure.)

Since the `location` version searches for FIPS codes, which are unique, specifying the state shouldn't be needed.

Reply:
Ahhh...  Yeah, that makes total sense. Thanks!